src/ipwndfu/main.py

```diff
@@ -159,16 +159,16 @@ def main():
         remove_alloc8()
 
     elif args.decrypt_gid:
-        decrypt_gid(device, args.decrypt_gid, match=args.match_device)
+        print(decrypt_gid(device, args.decrypt_gid, match=args.match_device))
```

Comment:
FUTURE IDEA:
Would be nice if this output machine readable format as well no?  Suppose you would do that by importing the module and having a reasonable API?

Reply:
That’s the plan. I originally started working on this because I was using ipwndfu as a module in a project for AES decryption.